<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/img2.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="EnergyCalc">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/image.png);
color:black;</string>
    </property>
    <property name="text">
     <string>Calculate H(r)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mu_value">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Energy_value">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mu_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>Reduced mass [mu]</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>x-coordinate</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>y-coordinate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="px_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="px_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>px-coordinate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jacobi_value">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mu_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>R3BP ORBITER CALCULATOR (C++ version)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mu_label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>Jacobi integral [J]</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string> py-coordinate is 
 computed via 
 the Jacobi integral</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="step_value">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="period_value">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="step_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>num. of steps per orbit</string>
    </property>
   </widget>
   <widget class="QLabel" name="period_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>numer of periods</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orbit_integration">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/img3.png);
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>RUN INTEGRATION</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="finish_run_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../example_C++_8/fig_1a.png</pixmap>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color:black;</string>
    </property>
    <property name="text">
     <string>Not show orbit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color:black;</string>
    </property>
    <property name="text">
     <string>Show orbit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mu_value</tabstop>
  <tabstop>x_value</tabstop>
  <tabstop>y_value</tabstop>
  <tabstop>px_value</tabstop>
  <tabstop>jacobi_value</tabstop>
  <tabstop>step_value</tabstop>
  <tabstop>period_value</tabstop>
  <tabstop>EnergyCalc</tabstop>
  <tabstop>orbit_integration</tabstop>
  <tabstop>Energy_value</tabstop>
  <tabstop>finish_run_label</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
